packages/block-editor/src/components/link-control/index.js

```diff
@@ -283,7 +294,9 @@ function LinkControl( {
 		<div
 			tabIndex={ -1 }
 			ref={ wrapperNode }
-			className="block-editor-link-control"
+			className={ classnames( 'block-editor-link-control', {
```

Comment:
The classname doesn't need to be added here, because it's already added at the top level of the interface skeleton.

Instead, I'd suggest re-structuring the CSS below so it takes advantage of that top-level classname, similarly to the [block toolbar CSS](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-toolbar/style.scss#L126).